lms/events/__init__.py

```diff
@@ -0,0 +1,66 @@
+from dataclasses import dataclass
+
+from pyramid.events import subscriber
+from pyramid.request import Request
+
+from lms.models import Event, EventType, LtiLaunches
+from lms.services import EventService, LTIRoleService
+
+
+@dataclass
+class LTILaunchEvent:
```

Comment:
I think we should keep the event type close to the subscriber function.

This could be lms/events/lti.py or something along those lines but this can be moved/split any way we want as it grows.